src/utils/api/http.js

```diff
@@ -37,7 +38,9 @@ const http = ({
         return response.json();
       });
   } catch (e) {
-    return Promise.reject(Error(e));
+    // eslint-disable-next-line no-console
+    console.error(e);
+    // return Promise.reject(Error(e));
```

Comment:
I have temporarily removed this promise rejection since its not being handled anyway. We have plans to introduce better error handling in #3485